blocks/banner/banner.js

```diff
@@ -0,0 +1,75 @@
+import {
+  div, button, h2, h3, form, input, a,
+} from '../../scripts/dom-helpers.js';
+
+function createLinkElement(hrefValue, innerHTML) {
+  const link = a({ class: 'spotlight-link', href: hrefValue }, innerHTML);
+
+  return link;
+}
+
+function createButton(hrefValue) {
+  const spotlightButton = button({
+    class: 'spotlight-button',
+    onclick: () => { window.location.href = hrefValue; },
+  }, 'LEARN MORE');
+
+  return spotlightButton;
+}
+
+function createBannerText(block) {
+  // extract text values from block elements
+  const spotlightTitle = block.querySelector('h2');
+  const spotlightSubtitle = block.querySelector('h3');
+
+  // create hyperlinks with the text values
+  const spotlightLink = block.querySelector('a');
+  const hrefValue = spotlightLink ? spotlightLink.href : '#';
+  const titleLink = createLinkElement(hrefValue, spotlightTitle.innerHTML);
+  const subtitleLink = createLinkElement(hrefValue, spotlightSubtitle.innerHTML);
+
+  spotlightTitle.remove();
+  spotlightSubtitle.remove();
+  spotlightLink.remove();
+
+  const heading = h2(titleLink);
+  const subheading = h3(subtitleLink);
+
+  // create the button
+  const spotlightButton = createButton(hrefValue);
```

Comment:
Not sure if this function provides any benefit as there's only one call to it.  Might has well put that code here.